vm/src/obj/objproperty.rs

```diff
@@ -252,6 +252,11 @@ pub struct PropertyBuilder<'a> {
     setter: Option<PyObjectRef>,
 }
 
+pub trait PropertySetterResult {}
+
+impl PropertySetterResult for PyResult {}
```

Comment:
Maybe this should just be `PyResult<()>`, as setters don't ever really return anything.

Reply:
That sounds better. I added another commit for that.